docs/sdk-configuration.md

```diff
@@ -72,6 +72,58 @@ You also need to mount the secret named after the `AWSIAMRole` resource into the
 
 See full [Java example project](https://github.com/mikkeloscar/kube-aws-iam-controller-java-example).
 
+## JS AWS SDK
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: aws-iam-js-example
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      application: aws-iam-js-example
+  template:
+    metadata:
+      labels:
+        application: aws-iam-js-example
+    spec:
+      containers:
+      - name: aws-iam-python-example
+        image: mikkeloscar/kube-aws-iam-controller-js-example:latest
+        env:
+        # must be set for the AWS SDK/AWS CLI to find the credentials file.
+        - name: AWS_SDK_LOAD_CONFIG
+          value: "true"
+        - name: AWS_SHARED_CREDENTIALS_FILE # used by python SDK
```

Comment:
nit: this is used by js SDK here :)

Reply:
Doh! Copy and paste oops there. Just fixed it!